_layouts/map.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+---
+<h2>{{ page.mapName }} v{{ page.version }}</h2>
+{% if page.mapCategory %}<h4>Category: {{ page.mapCategory }}</h4>{% endif %}
+{% if page.mapType == "MAP_SKIN" %}<h4>Type: Map Skin</h4>{% endif %}
+<img src="{{ page.img }}">
```

Comment:
I guess the schema I wrote for `triplea_maps.yaml` allows duplicate properties. 😞  I will investigate improving that...